src/Select/__tests__/Select.test.js

```diff
@@ -5,6 +5,8 @@ import { shallow } from "enzyme";
 
 import Select from "../Select";
 
+const mockChange = jest.fn();
+const placeholder = "Default placeholder";
 const objectOptions = [
```

Comment:
`visible` is not needed in this object more.

Reply:
Fixed